### ISSUE TITLE
Wrong REST api path used when the server address ends with a slash

### ISSUE THREAD

**Issue by markellus** [jensim/bitbucket_server_cli#27]
I noticed a bug where the tool does not work if you enter a server address with a slash like this:

```
.\bitbucket_server_cli.exe clone-projects -H
BitBucket server address: http://a-bitbucket-server.com:8080/
```

This will result in the following error message. As you can see, the REST api path is wrong ("/rest/api/1.0/" is doubled):

```
Failed fetching project repos. Failed fetching projects from bitbucket, status code: 404 Not Found.
Cause: Body: '{"message":"null for uri: http://a-bitbucket-server.com:8080/rest/api/1.0//rest/api/1.0/projects?limit=500&start=0","status-code":404}'
```

If I remove the slash behind the port number from the server address everything works as expected.

**Comment by jensim** [comment]
Thanks @markellus for raising this issue

**Comment by jensim** [comment]
Looks to me like you provided a server as `http://a-bitbucket-server.com:8080/rest/api/1.0/`
There was a bug that you was not able to provide `http://a-bitbucket-server.com:8080/` (an url with trailing slash), that I have now fixed in version `0.4.2`. You are still not able to use the /rest/api/1.0/ trailing uri, that will give you 404s
